From f58f08741356fc98a3a47099f39b40de74231310 Mon Sep 17 00:00:00 2001
From: "Steven Rostedt (Red Hat)" <rostedt@goodmis.org>
Date: Thu, 19 Jun 2014 09:40:09 -0400
Subject: [PATCH] Revert "drivers/net: tulip_remove_one needs to call
 pci_disable_device()"

This reverts commit 4a77edc195f0b03644e84dda00fcfe9827868e8e.

This commit was added to the -rt tree and submitted to stable. But because
the stable version inserted pci_disable_device() in a slightly different
location, when stable was merged it caused this to be duplicated.

Remove the version that was added to -rt and keep the stable one.

Link: http://lkml.kernel.org/r/1403149603.5189.10.camel@marge.simpson.net

Reported-by: Mike Galbraith <umgwanakikbuti@gmail.com>
Signed-off-by: Steven Rostedt <rostedt@goodmis.org>

diff --git a/drivers/net/ethernet/dec/tulip/tulip_core.c b/drivers/net/ethernet/dec/tulip/tulip_core.c
index a6d4653..7790160 100644
--- a/drivers/net/ethernet/dec/tulip/tulip_core.c
+++ b/drivers/net/ethernet/dec/tulip/tulip_core.c
@@ -1939,7 +1939,6 @@ static void tulip_remove_one(struct pci_dev *pdev)
 	pci_iounmap(pdev, tp->base_addr);
 	free_netdev (dev);
 	pci_release_regions (pdev);
-	pci_disable_device(pdev);
 	pci_set_drvdata (pdev, NULL);
 	pci_disable_device(pdev);
 
-- 
2.1.0

